README.md

```diff
@@ -42,7 +42,9 @@ Please see [The official documentation](https://www.vaultproject.io/docs/config/
 
 * `purge_config_dir`: Whether the `config_dir` should be purged before installing the generated config.
 
-* `install method`: Supports the values `repo` or `archive`. See [Installation parameters](#installation-parameters).
+* `install_method`: Supports the values `repo` or `archive`. See [Installation parameters](#installation-parameters).
+
+* `manage_unzip`: Whether to manage the `unzip` package on Debian/Ubuntu systems. (defaults to true if `install_method` is set to `archive`)
```

Comment:
I can remove it, if you insist, altough I think it would be better, since the archive type (.zip in this case) is also managed inside the module. It can be overriden using the `manage_unzip` parameters and using ensure_packages should prevent duplicate resource declarations.